code/tika/TikaRestClient.php

```diff
@@ -5,20 +5,58 @@
 
 class TikaRestClient extends Client
 {
+    /*
+    * @var Array
+    */
+    private static $_options = ['username' => null, 'password' => null];
+
+    /*
+    * @var Boolean
+    */
+    private static $_available = false;
+
+    /*
+    * @var Float
+    */
+    private static $_version = 0.0;
+
+    /*
+    * @var Array
+    */
+    protected $mimes = [];
+
+    public function __construct($baseUrl = '', $config = null)
+    {
+        if (defined('SS_TIKA_USERNAME') && defined('SS_TIKA_PASSWORD')) {
+            self::$_options = [
```

Comment:
maybe we should make this a non-static property?

Reply:
Agreed, updated